.gitignore

```diff
@@ -1,5 +1,7 @@
+.DS_Store
 /node_modules/
 /bin/
 /example/*
 !example/index.html
-.idea
\ No newline at end of file
+.idea
+/secret.sh
```

Comment:
Can you add explanation about the `secret.sh` file in the README?

Reply:
👍